autogen/main.tf.tmpl

```diff
@@ -117,6 +118,11 @@ resource "google_compute_backend_service" "default" {
     }
   }
 
+  log_config {
```

Comment:
We should make this block dynamic so it is only added if enable = true.

Reply:
It looks somehow if the block is not added, it will be enabled by default. So I think maybe still passing the value down from the config that consumes this module to the actual log_config block in the module might be better. And users have to write the log_config block anyway.

Reply:
Ack.